.github/workflows/blade-validate.yml

```diff
@@ -28,5 +28,6 @@ jobs:
       - name: Run Native Tests
         run: yarn test:react-native
         working-directory: packages/blade
-      - name: Run TypeScript Checks
-        run: yarn tsc
+      - name: Run TypeScript Checks Blade
```

Comment:
Do we need to specify "Blade" here? Since this is already running on blade repo 🤔

Reply:
Oh yeah we can remove it, We were checking blade & blade-old separately so thats why we named it like that.